src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java

```diff
@@ -55,6 +58,15 @@ public void testInvalidConfig() {
                 response.getBody().jsonPath().getString("errors[0]"));
     }
 
+    @Test
+    public void test51CharsFileName() {
+        Response response = postNewBundle(filenameWith51CharsJson);
+
+        assertEquals(200, response.getStatusCode());
```

Comment:
why do we return 200 if there's an error?

Reply:
This is current implementation where in the consuming service is expected to look at the errors before looking in to the body of the response.

Reply:
I will create separate JIRA ticket to chase this with the consuming services in terms of returning Error response code.